README.md

```diff
@@ -87,7 +87,7 @@ An `Entity` represents a singular existence of a particular Class, but does not
 
 When a new `Entity` is created, it contains no information. Once added to the [versioned store](https://github.com/Joystream/substrate-versioned-store-module), it returns a unique global identifier in the database, known as the `enitityId`. This identifier is an integer, counting from `1`. Despite having a global identity, it is still associated with a single `classId`.
 
-As the entity itself holds no data, one has to add schema support to it, and assign values to the properties available in the `in_schema_index`. These values must be in line with the property type, and its restrictions. If the property is "required", it must be populated. Properties in the `in_schema_index` that is not given a value will by default be given a `null` value. Note that properties properties that exists in the class (`in_class_index`), but not in the schema (`in_schema_index`) can not be populated. An entity can exist in multiple schemas simultaneously, but can not contain different values across `in_class_index` properties. Unlike a classes, schemas and properties, and entity can be updated, and be assigned new values.
+As the entity itself holds no data, one has to add schema support to it, and assign values to the properties available in the `in_schema_index`. These values must be in line with the property type, and its restrictions. If the property is "required", it must be populated. Properties in the `in_schema_index` that is not given a value will by default be given a `null` value. Note that properties that exist in the class (`in_class_index`), but not in the schema (`in_schema_index`) can not be populated. An entity can exist in multiple schemas simultaneously, but can not contain different values across `in_class_index` properties. Unlike classes, schemas, and properties, and entity can be updated, and be assigned new values.
```

Comment:
The part which states "Unlike classes, schemas, properties, and entity can be updated, and be assigned new values." seems pretty hard to read for me, because the "classes" have not been seperated very well from "schemas, properties, and entity". Do you think that it's easy for most people to understand?